### ISSUE TITLE
Bootstrap 5 Support

### ISSUE THREAD

**Issue by renebrenner** [KasperOlesen/DataTable-AltEditor#196]
Hello,

please add support for Bootstrap 5 or mention it in the release notes that it's not compatible.

**Comment by SinisterSpatula** [comment]
Yes please, it seems like every data-table-editor is for old versions of bootstrap, v2, v3, v4, etc, none have been updated to work with v5.  I'm ending up having to re-code my top navigation bar into bootstrap v3 compatible code, just to get a page with working data-table/alteditor and have a working nav bar that at least somewhat resembles the rest of my pages.  I'm very grateful that this project exists and that it's free, don't get me wrong, but man oh man wouldn't it be nice if it could work with newer bootstrap.
